media/js/firefox/sync.js

```diff
@@ -43,7 +42,6 @@
 
             if (fxMasterVersion >= 31) {
 
-                // Set syncCapable so we know not to call gaTrack()
```

Comment:
Should replace this comment with something, as `again later` on its own doesn't make much sense. :smile: